frontend/src/routes/(authenticated)/(dashboards)/admin/+page.ts

```diff
@@ -9,19 +9,14 @@ export async function load(event: PageLoadEvent) {
   const results = await client.query(graphql(`
         query loadAdminDashboard {
             projects(orderBy: [
-                {lastCommit: ASC_NULLS_FIRST},
+                {lastCommit: ASC},
```

Comment:
Maybe we don't even want nulls at the beginning. What if thre are 20 repos that never got an initial commit. I mean....we'd probably want to blow most of them away if there were so many, but...the init time is probably sufficient for sorting. We'd probably still want to be able to filter for null last-commets and make them visible.

Reply:
yeah we just decided at some point that we'd like this as the default, we hope to eventually support sorting on the client side but we're not there yet.